### ISSUE TITLE
508 compliance follow up work

### ISSUE THREAD

**Issue by KtejanKella** [HHS/simpler-grants-gov#10053]
### Summary

Testing engineers have provided feedback on ways to improve Section 508 compliance for the Opportunity Publishing Prototype (Opportunity List Page, Create Opportunity Page, Edit Opportunity Page). 

**Recommendation:** Install the [WAVE Evaluation Tool](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) (Chrome extension) to review 508 compliance

### Acceptance criteria
- [ ] Update label: Change “Opportunity #” → “Opportunity Number”
- [ ] Improve accessibility:
  - [ ] Ensure sufficient color contrast for text
  - [ ] Ensure sufficient color contrast for buttons
  - [ ] Pay special attention to red color usage
~~- [ ] Validation requirement: The Publish button should validate to ensure that "Publish Date" is set to the current date OR a date set in the future~~

**Comment by KtejanKella** [comment]
@brianlee-nava for your review

**Comment by brianlee-nava** [comment]
@KtejanKella updated the description on this ticket based on feedback from @aoysimmons-GH @Bhavna-Ramachandran.

**Comment by brianlee-nava** [comment]
@brianlee-nava CC: @KtejanKella Brian will remove last AC and put onto new ticket to keep scope smaller.

**Comment by brianlee-nava** [comment]
There is an assumption that our design patterns match USWDS. If that is not the case, what is missing? How do we prevent a similar issue in the future?

**Comment by desaiamit** [comment]
- Clarify the scope by listing the pages under Opportunity Publishing Prototype (Opportunity List, Opportunity Create, Opportunity Edit, ...)
- For AC - "Ensure sufficient color contrast for buttons" - The WAVE tool did not flag any color contrast issues on buttons. Which buttons (and corresponding page) are having contrast issues ?
- For AC - "Pay special attention to red color usage", what is expected here ?

**Comment by brianlee-nava** [comment]
@desaiamit tagging @aoysimmons-GH @glenblosser-nava to provide any additional context around contrast for buttons and use of red on the page.  I took a look by using the WAVE tool to check for accessibility issues. I also updated the scope above in the description. 

Opportunity List Page
- No contrast errors

Create Opportunity Page
- There are 3 contrast errors reported. All related to number of characters allowed. The font for that help text does not have enough contrast against the white background

<img width="1522" height="1196" alt="Image" src="https://github.com/user-attachments/assets/e871278c-f83b-4563-91c9-1f88f5794d7d" />

Edit Opportunity Page
- Text under Key information header: "Below is a summary of the key information for this opportunity. You can edit specific fields directly in this section."
  - really low contrast against white background

<img width="1208" height="666" alt="Image" src="https://github.com/user-attachments/assets/1be13e21-3473-4f9c-a5a9-225404dc4176" />

**Comment by aoysimmons-GH** [comment]
The button color contrast currently fails WCAG accessibility requirements. The hint is also not displayed in the summary.

@brianlee-nava , I'm not sure whether the client requires compliance with WCAG AA or WCAG AAA standards. Could you please confirm the expected accessibility level for this project?

below screenshot is example:

<img width="1022" height="866" alt="Image" src="https://github.com/user-attachments/assets/6897f155-a587-492b-b0d6-22339a4bd490" />

**Comment by brianlee-nava** [comment]
@aoysimmons-GH I just checked. The project targets WCAG 2.0 AA as its standard for Section 508 compliance.

**Comment by aoysimmons-GH** [comment]
Publish button

<img width="647" height="514" alt="Image" src="https://github.com/user-attachments/assets/f96600f4-0302-43ce-8598-3ccb2aec5940" />

**Comment by aoysimmons-GH** [comment]
Run run a check to the red validations. They are 508 compliant.

<img width="774" height="652" alt="Image" src="https://github.com/user-attachments/assets/babb4da3-33af-41ee-86f1-4b858954487d" />
